data/names/ru.json

```diff
@@ -3334,89 +3398,107 @@
       "Брэдшоу",
       "Брэй",
       "Брэкстон",
-      "Брэндон",
       "Брэнд",
+      "Брэндон",
+      "Брэннон",
       "Брэнтли",
-      "Брюстер",
       "Брюс",
+      "Брюстер",
       "Брюэр",
       "Будро",
       "Буй",
       "Букер",
+      "Булл",
       "Буллард",
       "Буллок",
-      "Булл",
       "Бун",
+      "Бургер",
       "Бургос",
       "Буржуа",
       "Буркетт",
       "Буркс",
       "Буркхарт",
+      "Бурлесон",
       "Бустаманте",
       "Бустос",
       "Бут",
+      "Буш",
       "Бушар",
       "Буше",
-      "Буш",
       "Буэно",
+      "Быстрый",
```

Comment:
This took me a moment. It's a direct translation of either Quick or, more likely, Swift. In the case of the latter, it would be transcribed Свифт, as in [Тейлор Свифт](https://ru.wikipedia.org/wiki/%D0%A1%D0%B2%D0%B8%D1%84%D1%82,_%D0%A2%D0%B5%D0%B9%D0%BB%D0%BE%D1%80), the famous singer, or [Джонатан Свифт](https://ru.wikipedia.org/wiki/%D0%A1%D0%B2%D0%B8%D1%84%D1%82,_%D0%94%D0%B6%D0%BE%D0%BD%D0%B0%D1%82%D0%B0%D0%BD), the famous writer.

Reply:
It's actually `Quick`. What should be the correct transcription?

Reply:
Either Квик or Куик then

Reply:
`Куик`, given that Jonathan Quick, the hockey player, is [Джонатан Куик](https://ru.wikipedia.org/wiki/%D0%9A%D1%83%D0%B8%D0%BA,_%D0%94%D0%B6%D0%BE%D0%BD%D0%B0%D1%82%D0%B0%D0%BD) on Wikipedia.

Reply:
[Мэтью Квик](https://ru.wikipedia.org/wiki/%D0%9A%D0%B2%D0%B8%D0%BA,_%D0%9C%D1%8D%D1%82%D1%8C%D1%8E)